### PR TITLE
docs(react-tooltip): Add info icon + tooltip story to Tooltip stories

### LINKED ISSUE #28602: Add info icon + tooltip example
Issue to track addition of icon + tooltip example in Tooltip's stories.

### PR DESCRIPTION
**Pull request by sopranopillow**
This PR adds a story to tooltip that shows how to build an info icon with a tooltip following an InfoIcon pattern.

## Related Issue(s)

- Fixes #28602

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-mount_base.html">84</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-mount.html">74</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/InfoButton-mount_base.html">10</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/InfoButton-mount.html">12</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Avatar-mount_base.html">657</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Avatar-mount.html">624</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Button-mount_base.html">306</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Button-mount.html">302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Field-mount_base.html">1111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Field-mount.html">1123</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProvider-mount_base.html">663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProvider-mount.html">671</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-mount_base.html">84</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-mount.html">74</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">65</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">71</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">60</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">77</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/InfoButton-mount_base.html">10</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/InfoButton-mount.html">12</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/MakeStyles-mount_base.html">858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/MakeStyles-mount.html">867</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Persona-mount_base.html">1694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/Persona-mount.html">1650</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/SpinButton-mount_base.html">1307</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28611/perf-test-react-components/SpinButton-mount.html">1354</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |   `84.992 kB`<br />`22.03 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `47.739 kB`<br />`14.521 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |   `15.695 kB`<br />`6.314 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `63.915 kB`<br />`19.001 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `67.576 kB`<br />`18.225 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `213.111 kB`<br />`59.12 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `36.409 kB`<br />`12.003 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |  `55.293 kB`<br />`16.535 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |     `6.48 kB`<br />`2.203 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `158.674 kB`<br />`43.399 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `44.692 kB`<br />`12.447 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `132.761 kB`<br />`34.005 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `78.385 kB`<br />`19.379 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |  `77.004 kB`<br />`18.981 kB` |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr>                                                              |   `34.171 kB`<br />`9.199 kB` |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr>                                                                                    |    `25.962 kB`<br />`8.49 kB` |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr>                                                                          |  `69.142 kB`<br />`20.294 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `47.463 kB`<br />`16.655 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/1082207b08959996cd736725735cadc01d0849b0'>1082207b08959996cd736725735cadc01d0849b0</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-tooltip/stories/Tooltip/TooltipIcon.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <Label>This is an icon with a Tooltip to show extra information</Label>
+      <Tooltip
+        content={{
+          children: 'Content must never contain focusable elements.',
+          id: contentId,
+        }}
+        positioning="above-start"
+        withArrow
+        relationship="label"
+        onVisibleChange={(e, { visible }) => setVisible(visible)}
```
Comment (khmakoto) [inline packages/react-components/react-tooltip/stories/Tooltip/TooltipIcon.stories.tsx:33]: The `visible` parameter here will clash with your definition on top

```suggestion
        onVisibleChange={(e, data) => setVisible(data.visible)}
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**